### DISCUSSION TITLE
Need help with Caub, network and settings blocked

### DISCUSSION THREAD

**Discussion by gomes-web** [3kh0/ext-remover#1020] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
When i do CAUB Prevent Updates my chrome://network#general is blocked and when I do CAUB Flags Prevent Updates my setting's app is also blocked by my organization, is there anyway to fix/go around this?

**Comment by akabutnicer** [comment, marked as answer]
Have you ever thought about just giving up? It's a worthless wack-a-mole game, you find an exploit, Chrome or your teacher patches or fixes it. If you are an insisting person, then why not try options like Skiovox or, what apparent developers have nicknamed (as I already discussed the exploit to Bypassi prior to its founding), "Corkey". There's no other way to do caub, unless you use  that newly created caubDNS, although I'm not exactly sure if it works, that's up to you to figure out. Anyways, this is a bad start for me arriving back to these discussions.

**Reply by AshtonDavies** [reply]
Maybe giving the "Corkey" nickname was a pretty dumb idea I admit, but it had no legitimate name (same for the "OP Crosh" title on vk6's Crosh & VMC exploit, and I was not a part of that one). I am not sure if you have a problem with such adaptations or not. _This is not an intended argument by any means, just an explanation._

@gomes-web You can use long-supported exploits/glitches if you want like @akabutnicer was saying, but a continuous effort just to be able to play games or lookup big oily men without notice is kinda pointless, as it gets to the point that the successful exploits are not very straightforward for everyone, which leads to more and more discussions.

To actually relate to your question, I am pretty sure there is nothing you can do about that if your OS settings are blocked by your administrator.

**Reply by akabutnicer** [reply]
What this guy said. *I need to take a break, first day back and I can't take this anymore*.
